.github/workflows/CI.yml

```diff
@@ -59,4 +59,5 @@ jobs:
     - name: load code and run tests
       shell: bash
       run: |
-        ros -e '(handler-bind (#+asdf3.2(asdf:bad-SYSTEM-NAME (function MUFFLE-WARNING))) (handler-case (ql:quickload :sento/tests) (error (a) (format t "caught error ~s~%~a~%" a a) (uiop:quit 1))))' -e "(unless (5am:run! 'senti.tests:test-suite) (uiop:quit 1))"
+        ros -e '(handler-bind (#+asdf3.2(asdf:bad-SYSTEM-NAME (function MUFFLE-WARNING))) (handler-case (ql:quickload :sento/tests) (serious-condition (a) (format t "caught error ~s~%~a~%" a a) (uiop:quit 1))))' \
+            -e '(handler-case (unless (5am:run! \'senti.tests:test-suite) (uiop:quit 1)) (serious-condtion (a) (format t "caught error ~s~%~a~%" a a) (uiop:quit 1)))'
```

Comment:
I suppose the issue is here: `senti` instead of `sento`. So it just doesn't find the test-suite.

But I think I'll do this as a separate PR/change.